test/model_recover.cc

```diff
@@ -26,7 +26,7 @@ class Model : public rabit::Serializable {
   }
 };
 
-inline void TestMax(Model *model, int ntrial, int iter) {
+inline void TestMax(Model *model, int iter) {
```

Comment:
can we also add integration tests to 
https://github.com/dmlc/rabit/blob/master/test/test.mk#L12

Reply:
What integration tests do you think should be added ?

Reply:
we should cover Allgather operation with mock=0,<op of allgather>,.. and see if recovery works properly.